package/yast2-network.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Fri Oct 21 11:01:04 UTC 2016 - kanderssen@suse.com
+- Use '2' as the default value for forward delay in bridges created
+  by the virt proposal (bsc#988278)
```

Comment:
The changes file describes the change being done, but it should say something like: preventing from doing this or that by ... doing. Or just an information about the change, not that you just changed 0 to 2 for unknown reason.